src/unitxt/inference.py

```diff
@@ -412,8 +411,7 @@ class WMLInferenceEngine(
         results = wml_inference.infer(dataset["test"])
     """
 
-    client: Any = None
-    credentials: Any = None
+    credentials: Optional[Dict[str, str]] = None
```

Comment:
```suggestion
    credentials: Optional[Dict[Literal["url", "apikey", "project_id"], str]] = None
```
To complement the docs